DEBUG 2022-03-17 23:33:48.135 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-17 23:33:48.135 UTC: 	 Running on: Linux lxplus791.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-17 23:33:48.135 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-17 23:33:48.136 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-17 23:33:48.136 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-17 23:33:48.137 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-17 23:33:48.140 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-17 23:33:48.141 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-17 23:33:48.141 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-17 23:33:48.144 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-18 00:09:30.105 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 00:09:30.105 UTC: 	 Running on: Linux lxplus791.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 00:09:30.105 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-18 00:09:30.106 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-18 00:09:30.107 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-18 00:09:30.108 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-18 00:09:30.109 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-18 00:09:30.109 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-18 00:09:30.109 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-18 00:09:30.116 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-18 00:35:10.997 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 00:35:10.997 UTC: 	 Running on: Linux lxplus791.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 00:35:10.997 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-18 00:35:10.998 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-18 00:35:10.998 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-18 00:35:10.999 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-18 00:35:11.004 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-18 00:35:11.004 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-18 00:35:11.005 UTC: 	 Caught ClientException exceptionhehe
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-18 00:35:11.007 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-18 00:36:08.014 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 00:36:08.014 UTC: 	 Running on: Linux lxplus791.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 00:36:08.014 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-18 00:36:08.015 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-18 00:36:08.016 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-18 00:36:08.016 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-18 00:36:08.016 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-18 00:36:08.017 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-18 00:36:08.017 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-18 00:36:08.019 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-18 09:40:45.105 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 09:40:45.105 UTC: 	 Running on: Linux lxplus721.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 09:40:45.105 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-18 09:40:45.105 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-18 09:40:45.109 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-18 09:40:45.109 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-18 09:40:45.114 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-18 09:40:45.116 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-18 09:40:45.116 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-18 09:40:45.123 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-18 09:57:30.077 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 09:57:30.077 UTC: 	 Running on: Linux lxplus721.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 09:57:30.077 UTC: 	 Executing command: 'checkwrite'
DEBUG 2022-03-18 09:57:30.079 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
DEBUG 2022-03-18 09:57:30.081 UTC: 	 Checking credentials
DEBUG 2022-03-18 09:57:30.081 UTC: 	 Found proxy file /tmp/x509up_u147761
DEBUG 2022-03-18 09:57:30.081 UTC: 	 Making sure it has proper VOMS extension
DEBUG 2022-03-18 09:57:30.081 UTC: 	 Executing command :
 voms-proxy-info --vo --file /tmp/x509up_u147761
DEBUG 2022-03-18 09:57:30.855 UTC: 	 output : cms

 error: 
 retcode : 0
DEBUG 2022-03-18 09:57:30.855 UTC: 	 Getting proxy life time left
DEBUG 2022-03-18 09:57:30.855 UTC: 	 Executing command :
 voms-proxy-info --actimeleft --timeleft --file /tmp/x509up_u147761
DEBUG 2022-03-18 09:57:31.500 UTC: 	 output : 689981
689981

 error: 
 retcode : 0
DEBUG 2022-03-18 09:57:31.500 UTC: 	 Proxy valid for 191:39:41 hours
DEBUG 2022-03-18 09:57:31.501 UTC: 	 Command use: checkwrite
DEBUG 2022-03-18 09:57:31.501 UTC: 	 Options use: ['--site=T2_CH_CSCS']
DEBUG 2022-03-18 09:57:31.501 UTC: 	 Instance is None
DEBUG 2022-03-18 09:57:31.501 UTC: 	 Server base url is None
DEBUG 2022-03-18 09:57:31.501 UTC: 	 Retrieving username ...
DEBUG 2022-03-18 09:57:31.501 UTC: 	 Retrieving username from CRIC...
DEBUG 2022-03-18 09:57:32.241 UTC: 	 username is jodedra
INFO 2022-03-18 09:57:32.241 UTC: 	 Will check write permission in the default location /store/user/<username>
INFO 2022-03-18 09:57:32.242 UTC: 	 Validating LFN /store/user/jodedra...
INFO 2022-03-18 09:57:32.242 UTC: 	 LFN /store/user/jodedra is valid.
INFO 2022-03-18 09:57:32.278 UTC: 	 Will use `gfal-copy`, `gfal-rm` commands for checking write permissions
INFO 2022-03-18 09:57:32.279 UTC: 	 Will check write permission in /store/user/jodedra on site T2_CH_CSCS
INFO 2022-03-18 09:57:34.775 UTC: 	 Will use PFN: rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/crab3checkwrite_20220318_105732.tmp
INFO 2022-03-18 09:57:34.776 UTC: 	 
Attempting to create (dummy) directory crab3checkwrite_20220318_105732 and copy (dummy) file crab3checkwrite_20220318_105732.tmp to /store/user/jodedra

INFO 2022-03-18 09:57:34.781 UTC: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-copy -p -v -t 180 file:///afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab3checkwrite_20220318_105732.tmp 'rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/crab3checkwrite_20220318_105732.tmp'
INFO 2022-03-18 09:57:34.781 UTC: 	 Please wait...
INFO 2022-03-18 09:57:35.391 UTC: 	 
Successfully created directory crab3checkwrite_20220318_105732 and copied file crab3checkwrite_20220318_105732.tmp to /store/user/jodedra
INFO 2022-03-18 09:57:35.392 UTC: 	 
Attempting to delete file rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/crab3checkwrite_20220318_105732.tmp

INFO 2022-03-18 09:57:35.392 UTC: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -v -t 180 'rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/crab3checkwrite_20220318_105732.tmp'
INFO 2022-03-18 09:57:35.392 UTC: 	 Please wait...
INFO 2022-03-18 09:57:35.826 UTC: 	 
Successfully deleted file rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/crab3checkwrite_20220318_105732.tmp
INFO 2022-03-18 09:57:35.827 UTC: 	 
Attempting to delete directory rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/

INFO 2022-03-18 09:57:35.827 UTC: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -r -v -t 180 'rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/'
INFO 2022-03-18 09:57:35.827 UTC: 	 Please wait...
INFO 2022-03-18 09:57:36.142 UTC: 	 
Successfully deleted directory rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220318_105732/
INFO 2022-03-18 09:57:36.209 UTC: 	 
Checkwrite Result:
INFO 2022-03-18 09:57:36.209 UTC: 	 Success: Able to write in /store/user/jodedra on site T2_CH_CSCS
DEBUG 2022-03-18 10:34:38.502 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 10:34:38.502 UTC: 	 Running on: Linux lxplus721.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 10:34:38.502 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-18 10:34:38.503 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-18 10:34:38.505 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-18 10:34:38.505 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-18 10:34:38.509 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-18 10:34:38.509 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-18 10:34:38.509 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-18 10:34:38.514 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-18 11:43:15.418 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 11:43:15.419 UTC: 	 Running on: Linux lxplus721.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 11:43:15.419 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-18 11:43:15.420 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-18 11:43:15.421 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-18 11:43:15.422 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-18 11:43:15.426 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-18 11:43:15.428 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-18 11:43:15.429 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-18 11:43:15.434 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-18 11:44:20.646 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-18 11:44:20.647 UTC: 	 Running on: Linux lxplus721.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-18 11:44:20.647 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-18 11:44:20.647 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-18 11:44:20.648 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-18 11:44:20.648 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-18 11:44:20.660 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-18 11:44:20.660 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-18 11:44:20.661 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW2dryrun/crab_JpsiK_20221603_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-18 11:44:20.663 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 09:56:03.553 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 09:56:03.554 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 09:56:03.554 UTC: 	 Executing command: 'status'
DEBUG 2022-03-21 09:56:03.554 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
ERROR 2022-03-21 09:56:03.555 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAWcomplete/crab_JpsiK_20221603_HLTRAW
ERROR 2022-03-21 09:56:03.555 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/status.py", line 48, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 275, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 646, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 508, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAWcomplete/crab_JpsiK_20221603_HLTRAW
INFO 2022-03-21 09:56:03.562 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:31:47.856 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:31:47.856 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:31:47.856 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-21 10:31:47.857 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-21 10:31:47.858 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-21 10:31:47.858 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-21 10:31:47.896 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-21 10:31:47.896 UTC: 	 Task requests 6000 MB of memory, above the allowed maximum of 5000 for a 1 core(s) job.

The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2022-03-21 10:31:47.897 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Task requests 6000 MB of memory, above the allowed maximum of 5000 for a 1 core(s) job.

The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2022-03-21 10:31:47.902 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:37:38.775 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:37:38.776 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:37:38.776 UTC: 	 Executing command: 'status'
DEBUG 2022-03-21 10:37:38.776 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
ERROR 2022-03-21 10:37:38.777 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW
ERROR 2022-03-21 10:37:38.777 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/status.py", line 48, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 275, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 646, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 508, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW
INFO 2022-03-21 10:37:38.783 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:37:57.167 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:37:57.168 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:37:57.168 UTC: 	 Executing command: 'tasks'
DEBUG 2022-03-21 10:37:57.168 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
DEBUG 2022-03-21 10:37:57.169 UTC: 	 Checking credentials
DEBUG 2022-03-21 10:37:57.169 UTC: 	 Found proxy file /tmp/x509up_u147761
DEBUG 2022-03-21 10:37:57.169 UTC: 	 Making sure it has proper VOMS extension
DEBUG 2022-03-21 10:37:57.169 UTC: 	 Executing command :
 voms-proxy-info --vo --file /tmp/x509up_u147761
DEBUG 2022-03-21 10:37:57.858 UTC: 	 output : cms

 error: 
 retcode : 0
DEBUG 2022-03-21 10:37:57.858 UTC: 	 Getting proxy life time left
DEBUG 2022-03-21 10:37:57.858 UTC: 	 Executing command :
 voms-proxy-info --actimeleft --timeleft --file /tmp/x509up_u147761
DEBUG 2022-03-21 10:37:58.481 UTC: 	 output : 689440
689440

 error: 
 retcode : 0
DEBUG 2022-03-21 10:37:58.481 UTC: 	 Proxy valid for 191:30:40 hours
DEBUG 2022-03-21 10:37:58.646 UTC: 	 Encoded data for curl request: subresource=delegatedn
DEBUG 2022-03-21 10:37:58.648 UTC: 	 Executing command :
 curl -v -X GET -H "User-Agent: CRABClient/v3.220210" -H "Accept: */*" --data @/tmp/crab_curlDataPR0RYi --cert "/tmp/x509up_u147761" --key "/tmp/x509up_u147761" --capath "/etc/grid-security/certificates/" "https://cmsweb.cern.ch:8443/crabserver/prod/info?subresource=delegatedn" | tee /dev/stderr 
DEBUG 2022-03-21 10:37:58.717 UTC: 	 output : {"result": [
 {"services": ["/DC=ch/DC=cern/OU=computers/CN=crab-(preprod|prod|dev)-tw(02|01).cern.ch|/DC=ch/DC=cern/OU=computers/CN=stefanov(m|m2).cern.ch|/DC=ch/DC=cern/OU=computers/CN=dciangot-tw.cern.ch"]}
]}

 error:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 188.185.89.202:8443...
* Connected to cmsweb.cern.ch (188.185.89.202) port 8443 (#0)
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/pki/tls/certs/ca-bundle.crt
  CApath: /etc/grid-security/certificates/
* TLSv1.2 (OUT), TLS header, Certificate Status (22):
} [5 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
} [512 bytes data]
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* TLSv1.2 (IN), TLS handshake, Server hello (2):
{ [109 bytes data]
* TLSv1.2 (IN), TLS handshake, Certificate (11):
{ [6199 bytes data]
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
{ [333 bytes data]
* TLSv1.2 (IN), TLS handshake, Request CERT (13):
{ [13495 bytes data]
* TLSv1.2 (IN), TLS handshake, Server finished (14):
{ [4 bytes data]
* TLSv1.2 (OUT), TLS handshake, Certificate (11):
} [6668 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
} [70 bytes data]
* TLSv1.2 (OUT), TLS handshake, CERT verify (15):
} [264 bytes data]
* TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
} [1 bytes data]
* TLSv1.2 (OUT), TLS handshake, Finished (20):
} [16 bytes data]
* TLSv1.2 (IN), TLS change cipher, Change cipher spec (1):
{ [1 bytes data]
* TLSv1.2 (IN), TLS handshake, Finished (20):
{ [16 bytes data]
* SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256
* ALPN, server accepted to use http/1.1
* Server certificate:
*  subject: DC=ch; DC=cern; OU=computers; CN=cmsweb.cern.ch
*  start date: Feb 17 13:05:12 2022 GMT
*  expire date: Mar 24 13:05:12 2023 GMT
*  subjectAltName: host "cmsweb.cern.ch" matched cert's "cmsweb.cern.ch"
*  issuer: DC=ch; DC=cern; CN=CERN Grid Certification Authority
*  SSL certificate verify ok.
} [5 bytes data]
> GET /crabserver/prod/info?subresource=delegatedn HTTP/1.1
> Host: cmsweb.cern.ch:8443
> User-Agent: CRABClient/v3.220210
> Accept: */*
> Content-Length: 22
> Content-Type: application/x-www-form-urlencoded
> 
} [22 bytes data]
* upload completely sent off: 22 out of 22 bytes
{ [5 bytes data]
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Mon, 21 Mar 2022 10:37:58 GMT
< Server: CherryPy/17.4.0
< Content-Type: application/json
< Vary: Accept
< Cache-Control: max-age=3600
< X-Rest-Status: 100
< Etag: "de92dcb56f87dabbe30965e60a433c3f1b873eab"
< Content-Length: 213
< X-Rest-Time: 1737.118 us
< CMS-Server-Time: D=7064 t=1647859078703277
< 
{ [5 bytes data]
100   235  100   213  100    22   5756    594 --:--:-- --:--:-- --:--:--  6184
* Connection #0 to host cmsweb.cern.ch left intact
{"result": [
 {"services": ["/DC=ch/DC=cern/OU=computers/CN=crab-(preprod|prod|dev)-tw(02|01).cern.ch|/DC=ch/DC=cern/OU=computers/CN=stefanov(m|m2).cern.ch|/DC=ch/DC=cern/OU=computers/CN=dciangot-tw.cern.ch"]}
]}

 retcode : 0
DEBUG 2022-03-21 10:37:58.719 UTC: 	 Registering user credentials on myproxy for /DC=ch/DC=cern/OU=computers/CN=crab-(preprod|prod|dev)-tw(02|01).cern.ch|/DC=ch/DC=cern/OU=computers/CN=stefanov(m|m2).cern.ch|/DC=ch/DC=cern/OU=computers/CN=dciangot-tw.cern.ch
DEBUG 2022-03-21 10:37:58.719 UTC: 	 Executing command :
 voms-proxy-info --identity --file /tmp/x509up_u147761
DEBUG 2022-03-21 10:37:59.385 UTC: 	 output : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra

 error: 
 retcode : 0
DEBUG 2022-03-21 10:37:59.386 UTC: 	 Getting myproxy life time left for 8b8ef16f353ac6386ac43a1dfe8c9a17c9742c0b
DEBUG 2022-03-21 10:37:59.386 UTC: 	 Executing command :
 myproxy-info -s myproxy.cern.ch -l 8b8ef16f353ac6386ac43a1dfe8c9a17c9742c0b
DEBUG 2022-03-21 10:37:59.733 UTC: 	 output : username: 8b8ef16f353ac6386ac43a1dfe8c9a17c9742c0b
owner: /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra
  renewal policy: /DC=ch/DC=cern/OU=computers/CN=crab-(preprod|prod|dev)-tw(02|01).cern.ch|/DC=ch/DC=cern/OU=computers/CN=stefanov(m|m2).cern.ch|/DC=ch/DC=cern/OU=computers/CN=dciangot-tw.cern.ch
  trusted retrieval policy: /DC=ch/DC=cern/OU=computers/CN=crab-(preprod|prod|dev)-tw(02|01).cern.ch|/DC=ch/DC=cern/OU=computers/CN=stefanov(m|m2).cern.ch|/DC=ch/DC=cern/OU=computers/CN=dciangot-tw.cern.ch
  timeleft: 576:10:08  (24.0 days)

 error: 
 retcode : 0
DEBUG 2022-03-21 10:37:59.733 UTC: 	 Myproxy is valid: 2074200
DEBUG 2022-03-21 10:37:59.733 UTC: 	 Result of myproxy credential check:
  Credential exists on myproxy: username: 8b8ef16f353ac6386ac43a1dfe8c9a17c9742c0b  - validity: 24 days, 0:10:00
DEBUG 2022-03-21 10:37:59.734 UTC: 	 Command use: tasks
DEBUG 2022-03-21 10:37:59.734 UTC: 	 Options use: []
DEBUG 2022-03-21 10:37:59.734 UTC: 	 Encoded data for curl request: subresource=version
DEBUG 2022-03-21 10:37:59.735 UTC: 	 Executing command :
 curl -v -X GET -H "User-Agent: CRABClient/v3.220210" -H "Accept: */*" --data @/tmp/crab_curlDatam5vYBe --cert "/tmp/x509up_u147761" --key "/tmp/x509up_u147761" --capath "/etc/grid-security/certificates/" "https://cmsweb.cern.ch:8443/crabserver/prod/info?subresource=version" | tee /dev/stderr 
DEBUG 2022-03-21 10:37:59.858 UTC: 	 output : {"result": [
 ["3.3.19", "3.3.20", "v3", "development", "py3.220218"]
]}

 error:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 188.185.89.202:8443...
* Connected to cmsweb.cern.ch (188.185.89.202) port 8443 (#0)
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/pki/tls/certs/ca-bundle.crt
  CApath: /etc/grid-security/certificates/
* TLSv1.2 (OUT), TLS header, Certificate Status (22):
} [5 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
} [512 bytes data]
* TLSv1.2 (IN), TLS handshake, Server hello (2):
{ [109 bytes data]
* TLSv1.2 (IN), TLS handshake, Certificate (11):
{ [6199 bytes data]
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
{ [333 bytes data]
* TLSv1.2 (IN), TLS handshake, Request CERT (13):
{ [13495 bytes data]
* TLSv1.2 (IN), TLS handshake, Server finished (14):
{ [4 bytes data]
* TLSv1.2 (OUT), TLS handshake, Certificate (11):
} [6668 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
} [70 bytes data]
* TLSv1.2 (OUT), TLS handshake, CERT verify (15):
} [264 bytes data]
* TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
} [1 bytes data]
* TLSv1.2 (OUT), TLS handshake, Finished (20):
} [16 bytes data]
* TLSv1.2 (IN), TLS change cipher, Change cipher spec (1):
{ [1 bytes data]
* TLSv1.2 (IN), TLS handshake, Finished (20):
{ [16 bytes data]
* SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256
* ALPN, server accepted to use http/1.1
* Server certificate:
*  subject: DC=ch; DC=cern; OU=computers; CN=cmsweb.cern.ch
*  start date: Feb 17 13:05:12 2022 GMT
*  expire date: Mar 24 13:05:12 2023 GMT
*  subjectAltName: host "cmsweb.cern.ch" matched cert's "cmsweb.cern.ch"
*  issuer: DC=ch; DC=cern; CN=CERN Grid Certification Authority
*  SSL certificate verify ok.
} [5 bytes data]
> GET /crabserver/prod/info?subresource=version HTTP/1.1
> Host: cmsweb.cern.ch:8443
> User-Agent: CRABClient/v3.220210
> Accept: */*
> Content-Length: 19
> Content-Type: application/x-www-form-urlencoded
> 
} [19 bytes data]
* upload completely sent off: 19 out of 19 bytes
{ [5 bytes data]
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Mon, 21 Mar 2022 10:37:59 GMT
< Server: CherryPy/17.4.0
< Content-Type: application/json
< Vary: Accept
< Cache-Control: max-age=3600
< X-Rest-Status: 100
< Etag: "b9e9d953ebda03e57a376ab381078e87e4cd4aa3"
< Content-Length: 73
< X-Rest-Time: 1711.845 us
< CMS-Server-Time: D=56833 t=1647859079795000
< 
{ [5 bytes data]
100    92  100    73  100    19    848    220 --:--:-- --:--:-- --:--:--  1069
* Connection #0 to host cmsweb.cern.ch left intact
{"result": [
 ["3.3.19", "3.3.20", "v3", "development", "py3.220218"]
]}

 retcode : 0
DEBUG 2022-03-21 10:37:59.859 UTC: 	 CRABClient version: v3.220210
DEBUG 2022-03-21 10:37:59.859 UTC: 	 Instance is prod
DEBUG 2022-03-21 10:37:59.859 UTC: 	 Server base url is cmsweb.cern.ch
DEBUG 2022-03-21 10:37:59.859 UTC: 	 Command api workflow
DEBUG 2022-03-21 10:37:59.859 UTC: 	 Encoded data for curl request: timestamp=2022-02-19
DEBUG 2022-03-21 10:37:59.860 UTC: 	 Executing command :
 curl -v -X GET -H "User-Agent: CRABClient/v3.220210" -H "Accept: */*" --data @/tmp/crab_curlDataNBo_jB --cert "/tmp/x509up_u147761" --key "/tmp/x509up_u147761" --capath "/etc/grid-security/certificates/" "https://cmsweb.cern.ch:8443/crabserver/prod/workflow?timestamp=2022-02-19" | tee /dev/stderr 
DEBUG 2022-03-21 10:37:59.976 UTC: 	 output : {"desc": {"columns": ["tm_taskname", "tm_task_status", "tw_name", "tm_user_dn"]}, "result": [
 ["220315_104816:jodedra_crab_JpsiK_20220315", "SUBMITTED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220316_132428:jodedra_crab_JpsiK_20221603_HLTRAW", "KILLED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220317_233434:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_001042:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_124644:jodedra_crab_JpsiK_20221603_HLTRAW", "KILLED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_121342:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_001728:jodedra_crab_JpsiK_20220315", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_003704:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_094214:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_000030:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_103527:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_114502:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220317_231348:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
]}

 error:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 188.185.89.202:8443...
* Connected to cmsweb.cern.ch (188.185.89.202) port 8443 (#0)
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/pki/tls/certs/ca-bundle.crt
  CApath: /etc/grid-security/certificates/
* TLSv1.2 (OUT), TLS header, Certificate Status (22):
} [5 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
} [512 bytes data]
* TLSv1.2 (IN), TLS handshake, Server hello (2):
{ [109 bytes data]
* TLSv1.2 (IN), TLS handshake, Certificate (11):
{ [6199 bytes data]
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
{ [333 bytes data]
* TLSv1.2 (IN), TLS handshake, Request CERT (13):
{ [13495 bytes data]
* TLSv1.2 (IN), TLS handshake, Server finished (14):
{ [4 bytes data]
* TLSv1.2 (OUT), TLS handshake, Certificate (11):
} [6668 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
} [70 bytes data]
* TLSv1.2 (OUT), TLS handshake, CERT verify (15):
} [264 bytes data]
* TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
} [1 bytes data]
* TLSv1.2 (OUT), TLS handshake, Finished (20):
} [16 bytes data]
* TLSv1.2 (IN), TLS change cipher, Change cipher spec (1):
{ [1 bytes data]
* TLSv1.2 (IN), TLS handshake, Finished (20):
{ [16 bytes data]
* SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256
* ALPN, server accepted to use http/1.1
* Server certificate:
*  subject: DC=ch; DC=cern; OU=computers; CN=cmsweb.cern.ch
*  start date: Feb 17 13:05:12 2022 GMT
*  expire date: Mar 24 13:05:12 2023 GMT
*  subjectAltName: host "cmsweb.cern.ch" matched cert's "cmsweb.cern.ch"
*  issuer: DC=ch; DC=cern; CN=CERN Grid Certification Authority
*  SSL certificate verify ok.
} [5 bytes data]
> GET /crabserver/prod/workflow?timestamp=2022-02-19 HTTP/1.1
> Host: cmsweb.cern.ch:8443
> User-Agent: CRABClient/v3.220210
> Accept: */*
> Content-Length: 20
> Content-Type: application/x-www-form-urlencoded
> 
} [20 bytes data]
* upload completely sent off: 20 out of 20 bytes
{ [5 bytes data]
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Mon, 21 Mar 2022 10:37:59 GMT
< Server: CherryPy/17.4.0
< Content-Type: application/json
< Vary: Accept
< Cache-Control: max-age=3600
< X-Rest-Status: 100
< Etag: "1e7a70a95d8797bf879fe7c782e29a958aa907d9"
< Content-Length: 2199
< X-Rest-Time: 46473.980 us
< CMS-Server-Time: D=52777 t=1647859079917329
< 
{ [5 bytes data]
100  2219  100  2199  100    20  26178    238 --:--:-- --:--:-- --:--:-- 26416
* Connection #0 to host cmsweb.cern.ch left intact
{"desc": {"columns": ["tm_taskname", "tm_task_status", "tw_name", "tm_user_dn"]}, "result": [
 ["220315_104816:jodedra_crab_JpsiK_20220315", "SUBMITTED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220316_132428:jodedra_crab_JpsiK_20221603_HLTRAW", "KILLED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220317_233434:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_001042:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_124644:jodedra_crab_JpsiK_20221603_HLTRAW", "KILLED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_121342:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_001728:jodedra_crab_JpsiK_20220315", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_003704:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_094214:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_000030:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_103527:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220318_114502:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
,["220317_231348:jodedra_crab_JpsiK_20221603_HLTRAW", "UPLOADED", "crab-prod-tw01", "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jodedra/CN=857621/CN=Jay Odedra"]
]}

 retcode : 0
INFO 2022-03-21 10:37:59.977 UTC: 	 
List of tasks from 2022-02-19 until 2022-03-21
INFO 2022-03-21 10:37:59.977 UTC: 	 Beware that STATUS here does not include information from grid jobs
INFO 2022-03-21 10:37:59.977 UTC: 	 ================================================================================
INFO 2022-03-21 10:37:59.977 UTC: 	 NAME								STATUS
INFO 2022-03-21 10:37:59.977 UTC: 	 ================================================================================
INFO 2022-03-21 10:37:59.977 UTC: 	 220318_124644:jodedra_crab_JpsiK_20221603_HLTRAW
								KILLED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_121342:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_114502:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_103527:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_094214:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_003704:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_001728:jodedra_crab_JpsiK_20220315
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_001042:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220318_000030:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.978 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.978 UTC: 	 220317_233434:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.979 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.979 UTC: 	 220317_231348:jodedra_crab_JpsiK_20221603_HLTRAW
								UPLOADED
INFO 2022-03-21 10:37:59.979 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.979 UTC: 	 220316_132428:jodedra_crab_JpsiK_20221603_HLTRAW
								KILLED
INFO 2022-03-21 10:37:59.979 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.979 UTC: 	 220315_104816:jodedra_crab_JpsiK_20220315
								SUBMITTED
INFO 2022-03-21 10:37:59.979 UTC: 	 --------------------------------------------------------------------------------
INFO 2022-03-21 10:37:59.979 UTC: 	 

INFO 2022-03-21 10:37:59.982 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:38:27.798 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:38:27.799 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:38:27.799 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-21 10:38:27.799 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-21 10:38:27.800 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-21 10:38:27.800 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-21 10:38:27.840 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-21 10:38:27.841 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-21 10:38:27.842 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-21 10:38:27.848 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:39:51.934 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:39:51.934 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:39:51.935 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-21 10:39:51.935 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-21 10:39:51.936 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-21 10:39:51.936 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-21 10:39:51.944 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-21 10:39:51.944 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-21 10:39:51.944 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-21 10:39:51.947 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:42:09.551 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:42:09.552 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:42:09.552 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-21 10:42:09.552 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-21 10:42:09.553 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-21 10:42:09.553 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-21 10:42:09.554 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-21 10:42:09.554 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-21 10:42:09.554 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-21 10:42:09.557 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:45:07.744 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:45:07.744 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:45:07.744 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-21 10:45:07.744 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-21 10:45:07.745 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-21 10:45:07.746 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-21 10:45:07.746 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-21 10:45:07.746 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-21 10:45:07.747 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-21 10:45:07.754 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-21 10:46:29.453 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-21 10:46:29.453 UTC: 	 Running on: Linux lxplus708.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-21 10:46:29.453 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-21 10:46:29.453 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-21 10:46:29.454 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-21 10:46:29.454 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-21 10:46:29.459 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-21 10:46:29.459 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-21 10:46:29.459 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW21032022/crab_JpsiK_20222103_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-21 10:46:29.462 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 11:55:36.394 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 11:55:36.395 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 11:55:36.395 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 11:55:36.395 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 11:55:36.397 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 11:55:36.397 UTC: 	 Loading CRAB configuration file.
INFO 2022-03-25 11:55:36.404 UTC: 	 Wrong format in command-line argument '--dryrun'. Expected format is <section-name>.<parameter-name>=<parameter-value>.
INFO 2022-03-25 11:55:36.404 UTC: 	 If the argument '--dryrun' is an option to the submit command, try 'crab submit --dryrun [value for --dryrun option (if required)] [arguments]'.
ERROR 2022-03-25 11:55:36.404 UTC: 	 ERROR: Wrong command-line format.
ERROR 2022-03-25 11:55:36.404 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 67, in loadConfig
    raise ConfigurationException("ERROR: Wrong command-line format.")
ConfigurationException: ERROR: Wrong command-line format.
INFO 2022-03-25 11:55:36.415 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 13:05:12.370 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 13:05:12.371 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 13:05:12.371 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 13:05:12.371 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 13:05:12.373 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 13:05:12.374 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-25 13:05:12.377 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-25 13:05:12.377 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-25 13:05:12.377 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-25 13:05:12.386 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 13:32:57.854 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 13:32:57.855 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 13:32:57.855 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 13:32:57.856 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 13:32:57.859 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 13:32:57.859 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-25 13:32:57.864 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-25 13:32:57.864 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun2/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-25 13:32:57.864 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun2/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-25 13:32:57.873 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 13:47:00.573 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 13:47:00.574 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 13:47:00.574 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 13:47:00.574 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 13:47:00.575 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 13:47:00.575 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-25 13:47:00.586 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-25 13:47:00.586 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun2/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-25 13:47:00.586 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun2/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-25 13:47:00.590 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 14:12:08.967 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 14:12:08.968 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 14:12:08.968 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 14:12:08.968 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 14:12:08.970 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 14:12:08.970 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-25 14:12:08.996 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-25 14:12:08.997 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun4/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-25 14:12:08.999 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun4/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-25 14:12:09.015 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 14:16:05.054 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 14:16:05.055 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 14:16:05.055 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 14:16:05.055 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 14:16:05.056 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 14:16:05.056 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-25 14:16:05.057 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-25 14:16:05.057 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun5/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-25 14:16:05.057 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun5/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-25 14:16:05.060 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 14:27:25.193 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 14:27:25.194 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 14:27:25.194 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 14:27:25.194 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 14:27:25.196 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 14:27:25.196 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-25 14:27:25.198 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-25 14:27:25.198 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun6/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-25 14:27:25.198 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun6/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-25 14:27:25.202 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 14:29:25.984 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 14:29:25.985 UTC: 	 Running on: Linux lxplus718.cern.ch 3.10.0-1160.53.1.el7.x86_64 #1 SMP Fri Jan 14 13:59:45 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 14:29:25.985 UTC: 	 Executing command: 'submit'
DEBUG 2022-03-25 14:29:25.985 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
INFO 2022-03-25 14:29:25.986 UTC: 	 Will use CRAB configuration file crab_step2.py
DEBUG 2022-03-25 14:29:25.986 UTC: 	 Loading CRAB configuration file.
DEBUG 2022-03-25 14:29:25.986 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2022-03-25 14:29:25.987 UTC: 	 Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun7/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
ERROR 2022-03-25 14:29:25.987 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022dryrun7/crab_JpsiK_20222503_HLTRAW' already exists 
Please change the requestName in the config file
INFO 2022-03-25 14:29:25.989 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 15:39:26.719 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 15:39:26.719 UTC: 	 Running on: Linux lxplus752.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 15:39:26.719 UTC: 	 Executing command: 'checkusername'
DEBUG 2022-03-25 15:39:26.719 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
DEBUG 2022-03-25 15:39:26.721 UTC: 	 Checking credentials
DEBUG 2022-03-25 15:39:26.721 UTC: 	 Found proxy file /afs/cern.ch/user/j/jodedra/private/x509up_u147761
DEBUG 2022-03-25 15:39:26.721 UTC: 	 Making sure it has proper VOMS extension
DEBUG 2022-03-25 15:39:26.721 UTC: 	 Executing command :
 voms-proxy-info --vo --file /afs/cern.ch/user/j/jodedra/private/x509up_u147761
DEBUG 2022-03-25 15:39:28.161 UTC: 	 output : cms

 error: 
 retcode : 0
DEBUG 2022-03-25 15:39:28.161 UTC: 	 Getting proxy life time left
DEBUG 2022-03-25 15:39:28.161 UTC: 	 Executing command :
 voms-proxy-info --actimeleft --timeleft --file /afs/cern.ch/user/j/jodedra/private/x509up_u147761
DEBUG 2022-03-25 15:39:29.503 UTC: 	 output : 690607
690607

 error: 
 retcode : 0
DEBUG 2022-03-25 15:39:29.503 UTC: 	 Proxy valid for 191:50:07 hours
DEBUG 2022-03-25 15:39:29.503 UTC: 	 Command use: checkusername
DEBUG 2022-03-25 15:39:29.503 UTC: 	 Options use: []
DEBUG 2022-03-25 15:39:29.503 UTC: 	 Instance is None
DEBUG 2022-03-25 15:39:29.503 UTC: 	 Server base url is None
DEBUG 2022-03-25 15:39:29.503 UTC: 	 Retrieving username ...
DEBUG 2022-03-25 15:39:29.503 UTC: 	 Retrieving username from CRIC...
DEBUG 2022-03-25 15:39:30.479 UTC: 	 username is jodedra
INFO 2022-03-25 15:39:30.479 UTC: 	 Username is: jodedra
DEBUG 2022-03-25 15:40:14.167 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 15:40:14.167 UTC: 	 Running on: Linux lxplus752.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 15:40:14.168 UTC: 	 Executing command: 'checkwrite'
DEBUG 2022-03-25 15:40:14.168 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
DEBUG 2022-03-25 15:40:14.169 UTC: 	 Checking credentials
DEBUG 2022-03-25 15:40:14.169 UTC: 	 Found proxy file /afs/cern.ch/user/j/jodedra/private/x509up_u147761
DEBUG 2022-03-25 15:40:14.169 UTC: 	 Making sure it has proper VOMS extension
DEBUG 2022-03-25 15:40:14.169 UTC: 	 Executing command :
 voms-proxy-info --vo --file /afs/cern.ch/user/j/jodedra/private/x509up_u147761
DEBUG 2022-03-25 15:40:15.195 UTC: 	 output : cms

 error: 
 retcode : 0
DEBUG 2022-03-25 15:40:15.195 UTC: 	 Getting proxy life time left
DEBUG 2022-03-25 15:40:15.195 UTC: 	 Executing command :
 voms-proxy-info --actimeleft --timeleft --file /afs/cern.ch/user/j/jodedra/private/x509up_u147761
DEBUG 2022-03-25 15:40:16.107 UTC: 	 output : 690560
690560

 error: 
 retcode : 0
DEBUG 2022-03-25 15:40:16.108 UTC: 	 Proxy valid for 191:49:20 hours
DEBUG 2022-03-25 15:40:16.108 UTC: 	 Command use: checkwrite
DEBUG 2022-03-25 15:40:16.108 UTC: 	 Options use: ['--site=T2_CH_CSCS']
DEBUG 2022-03-25 15:40:16.108 UTC: 	 Instance is None
DEBUG 2022-03-25 15:40:16.108 UTC: 	 Server base url is None
DEBUG 2022-03-25 15:40:16.108 UTC: 	 Retrieving username ...
DEBUG 2022-03-25 15:40:16.108 UTC: 	 Retrieving username from CRIC...
DEBUG 2022-03-25 15:40:16.901 UTC: 	 username is jodedra
INFO 2022-03-25 15:40:16.901 UTC: 	 Will check write permission in the default location /store/user/<username>
INFO 2022-03-25 15:40:16.902 UTC: 	 Validating LFN /store/user/jodedra...
INFO 2022-03-25 15:40:16.902 UTC: 	 LFN /store/user/jodedra is valid.
INFO 2022-03-25 15:40:16.992 UTC: 	 Will use `gfal-copy`, `gfal-rm` commands for checking write permissions
INFO 2022-03-25 15:40:16.992 UTC: 	 Will check write permission in /store/user/jodedra on site T2_CH_CSCS
INFO 2022-03-25 15:40:21.698 UTC: 	 Will use PFN: rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/crab3checkwrite_20220325_164016.tmp
INFO 2022-03-25 15:40:21.699 UTC: 	 
Attempting to create (dummy) directory crab3checkwrite_20220325_164016 and copy (dummy) file crab3checkwrite_20220325_164016.tmp to /store/user/jodedra

INFO 2022-03-25 15:40:21.707 UTC: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-copy -p -v -t 180 file:///afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab3checkwrite_20220325_164016.tmp 'rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/crab3checkwrite_20220325_164016.tmp'
INFO 2022-03-25 15:40:21.708 UTC: 	 Please wait...
INFO 2022-03-25 15:40:22.709 UTC: 	 
Successfully created directory crab3checkwrite_20220325_164016 and copied file crab3checkwrite_20220325_164016.tmp to /store/user/jodedra
INFO 2022-03-25 15:40:22.709 UTC: 	 
Attempting to delete file rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/crab3checkwrite_20220325_164016.tmp

INFO 2022-03-25 15:40:22.709 UTC: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -v -t 180 'rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/crab3checkwrite_20220325_164016.tmp'
INFO 2022-03-25 15:40:22.709 UTC: 	 Please wait...
INFO 2022-03-25 15:40:23.509 UTC: 	 
Successfully deleted file rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/crab3checkwrite_20220325_164016.tmp
INFO 2022-03-25 15:40:23.509 UTC: 	 
Attempting to delete directory rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/

INFO 2022-03-25 15:40:23.510 UTC: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -r -v -t 180 'rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/'
INFO 2022-03-25 15:40:23.510 UTC: 	 Please wait...
INFO 2022-03-25 15:40:24.000 UTC: 	 
Successfully deleted directory rucio 1.27.2
davs://storage01.lcg.cscs.ch:2880//pnfs/lcg.cscs.ch/cms/trivcat/store/user/jodedra/crab3checkwrite_20220325_164016/
INFO 2022-03-25 15:40:24.061 UTC: 	 
Checkwrite Result:
INFO 2022-03-25 15:40:24.061 UTC: 	 Success: Able to write in /store/user/jodedra on site T2_CH_CSCS
DEBUG 2022-03-25 16:02:23.469 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 16:02:23.470 UTC: 	 Running on: Linux lxplus752.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 16:02:23.470 UTC: 	 Executing command: 'status'
DEBUG 2022-03-25 16:02:23.470 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
ERROR 2022-03-25 16:02:23.471 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022submittedcheckfails
ERROR 2022-03-25 16:02:23.471 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/status.py", line 48, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 275, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 646, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 508, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022submittedcheckfails
INFO 2022-03-25 16:02:23.481 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
DEBUG 2022-03-25 16:02:39.722 UTC: 	 CRAB Client version: v3.220210
DEBUG 2022-03-25 16:02:39.723 UTC: 	 Running on: Linux lxplus752.cern.ch 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2022-03-25 16:02:39.723 UTC: 	 Executing command: 'status'
DEBUG 2022-03-25 16:02:39.723 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jodedra/.crab3
ERROR 2022-03-25 16:02:39.724 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022submittedcheckfails
ERROR 2022-03-25 16:02:39.724 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 162, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/bin/crab.py", line 147, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/status.py", line 48, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 275, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 646, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.220210.00/lib/CRABClient/Commands/SubCommand.py", line 508, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab_HLTRAW25032022submittedcheckfails
INFO 2022-03-25 16:02:39.727 UTC: 	 Log file is /afs/cern.ch/user/j/jodedra/CMSSW_11_2_2_patch1/src/generatorstuff/Run3TriggerPerf-1/samples/mc/crab.log
